.golangci.yml

```diff
@@ -89,8 +89,6 @@ linters-settings:
         deny:
           - pkg: "cosmossdk.io/errors"
             desc: Use the standard library instead
-          - pkg: "github.com/ethereum/go-ethereum"
```

Comment:
Why do we need to break this rule? Is every other chain family going to show up?

Reply:
Yes, there will be other chain-specific encoders in `reportcodecs`